docs/ops/image/Interpolate_4.md

```diff
@@ -0,0 +1,454 @@
+## Interpolate <a name="Interpolate"></a>
+
+**Versioned name**: *Interpolate-4*
+
+**Category**: Image processing
+
+**Short description**: *Interpolate* layer performs interpolation of independent slices in input tensor by specified dimensions and attributes.
+
+**Attributes**
+
+* *axes*
+
+  * **Description**: `axes` specify dimension indices where interpolation is applied, and `axes` is any unordered list of indices of different dimensions of input tensor, e.g. `[0, 4]`, `[4, 0]`, `[4, 2, 1]`, `[1, 2, 3]`. These indices should be non-negative integers from `0` to `rank(data) - 1` inclusively.  Other dimensions do not change. The order of elements in `axes` attribute matters, and mapped directly to elements in the 2nd input `target_spatial_shape`. Namely, `output_shape[axes[i]] = target_spatial_shape[i]` for all `i in range(0, len(axes))` and `output_shape[j] = input_shape[j] + pads_begin[j] + pads_end[j]` for `j not in axes`, `j in range(0, rank(data))`.
+  * **Range of values**: list of non-negative integer numbers
+  * **Type**: `int[]`
+  * **Default value**: None
+  * **Required**: *yes*
+
+* *mode*
+
+  * **Description**: specifies type of interpolation
+  * **Range of values**: one of `nearest`, `linear`, `linear_onnx`, `cubic`, `area`
+  * **Type**: string
+  * **Default value**: none
+  * **Required**: *yes*
+
+* *coordinate_transformation_mode*
+
+  * **Description**: specifies how to transform the coordinate in the resized tensor to the coordinate in the original tensor
+  * **Range of values**: one of `half_pixel`, `pytorch_half_pixel`, `asymmetric`, `tf_half_pixel_for_nn`, `align_corners`
```

Comment:
Please, add verbal description of possible values.

Reply:
Added.